src/platform/site-wide/sass/shame.scss

```diff
@@ -581,3 +581,26 @@ img[data-srcset] {
 .va-u-outline--none:focus {
   outline: none;
 }
+
+.medium-screen-vads-u-border-left-radius--0 {
+  @media (min-width: $medium-screen) {
+    border-bottom-left-radius: 0;
+    border-top-left-radius: 0;
+  }
+}
+
+.medium-screen-vads-u-background-color--white {
+  @media (min-width: $medium-screen) {
+    background-color: #ffffff !important;
+  }
+}
+
+.vads-u-border-bottom-radius--0 {
```

Comment:
Should this go into formation directly instead of into this sitewide css? cc @cvalarida

Reply:
Yeah, if it's `.vads*`, it should be in [`formation`](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/utilities/_border.scss), not in `shame.scss`.

Reply:
Are utilities for `border-radius` something we're okay with adding into Formation? 

I was kinda thinking we could start with just using `va-` as the namespace, then once the utilities are added into Formation, we can revisit this code by deleting SCSS and replacing classes with the `vads-` version. That way the `vads-` version is still reserved for Formation classes, not vets-website, so if we end up never being able to add them to Formation then at least we didn't borrow the namespace.

Reply:
My position is that sitewide css should always be in the design system since it's the only place where there's a process for managing css.